specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ApplicationOperationResults.json

```diff
@@ -0,0 +1,188 @@
+{
+    "swagger": "2.0",
```

Comment:
New swagger files has to be referenced from `readme.md`

Reply:
Done